tools/setup/generate-fixtures

```diff
@@ -5,12 +5,29 @@ function migration_status {
     ./manage.py get_migration_status --settings=zproject.test_settings > $1
 }
 
-if  [ "$1" != "--force" ]; then
+force=0
+backend=0
+
+if [ "$1" == "--backend" ]; then
+    backend=1
+elif [ "$1" ==  "--force" ]; then
+    force=1
+fi
+
+if [ "$2" == "--backend" ]; then
+    backend=1
+elif [ "$2" ==  "--force" ]; then
+    force=1
+fi
+
+if  [ $force -eq 0 ]; then
     "$(dirname "$0")/../../scripts/setup/terminate-psql-sessions" zulip zulip_test zulip_test_base zulip_test_template
-    psql -h localhost postgres zulip_test << EOF
+    if [ $backend -eq 0 ]; then
+        psql -h localhost postgres zulip_test << EOF
 DROP DATABASE IF EXISTS zulip_test;
 CREATE DATABASE zulip_test TEMPLATE zulip_test_template;
 EOF
+    fi
     sh "$(dirname "$0")/../../scripts/setup/flush-memcached"
```

Comment:
I think this entire block is not require if `backend = 0`, right?  Like, no need to terminate psql sessions or flush memcached either.

Actually ... do we even need to call `generate-fixtures` without `--force` from the backend tests?  It seems like we probably don't, if the backend tests are doing their own database creation/deletion.